### ISSUE TITLE
Need to add a track prop called `alignment` 

### ISSUE THREAD

**Issue by sehilyi** [gosling-lang/gosling.js#295]
**Basic Idea:**
```ts
{
   ..., // common specs
   alignment: 'overlay' or 'stack',
   tracks: [
        {...}, // track definition
        { alignment: 'overlay', tracks: [...] }, // another a group of track definition
   ]
}
```

This way, view and track definitions are more consistent.

**Comment by sehilyi** [comment]
**To-Do**
- [ ] Visualization's title should be differentiated from the track's title
```ts
{
   title: 'TITLE', // Currently, this is both the visualization title and the track title.
   alignment: 'overlay',
   tracks: [{...}, ...]
}
```

**Comment by sehilyi** [comment]
Closing this since this is addressed by #319. 

We would need to think about how to separate properties that are used internally, like `overlay: []` and `overlayOnPreviousTrack`, from the properties that are exposed to users.
